
Build system updates. Imported semi-automatically from .diff.gz.

diff --git a/Makefile.am b/Makefile.am
index 1ff290e..95143fb 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -32,12 +32,6 @@ extra =
 endif
 lib_LTLIBRARIES = libgc.la $(extra) 
 
-include_HEADERS = include/gc.h include/gc_local_alloc.h \
-include/gc_pthread_redirects.h include/gc_config_macros.h \
-include/leak_detector.h include/gc_typed.h @addincludes@
-
-EXTRA_HEADERS = include/gc_cpp.h include/gc_allocator.h
-
 if POWERPC_DARWIN
 asm_libgc_sources = powerpc_darwin_mach_dep.s
 else
@@ -65,7 +59,7 @@ EXTRA_libgc_la_SOURCES = alpha_mach_dep.S \
     sparc_sunos4_mach_dep.s ia64_save_regs_in_stack.s
 
 libgccpp_la_SOURCES = gc_cpp.cc
-libgccpp_la_LIBADD = $(THREADDLLIBS) $(UNWINDLIBS)
+libgccpp_la_LIBADD = $(THREADDLLIBS) $(UNWINDLIBS) libgc.la
 libgccpp_la_LDFLAGS = -version-info 1:2:0
 
 EXTRA_DIST += alpha_mach_dep.S mips_sgi_mach_dep.s sparc_mach_dep.S
@@ -135,11 +129,6 @@ dist_noinst_SCRIPTS = callprocs configure.host
     ## callprocs --- used by Makefile.{dj,direct}
     ## configure.host --- used by Makefile.{am,dj,direct}
 
-# headers which are not installed
-# (see include/Makefile.am for more)
-#
-dist_noinst_HEADERS = version.h
-
 # documentation which is not installed
 #
 EXTRA_DIST += README.QUICK
diff --git a/acinclude.m4 b/acinclude.m4
index 72602a0..db6950e 100644
--- a/acinclude.m4
+++ b/acinclude.m4
@@ -17,7 +17,7 @@
 # [0-9]+[.][0-9]+(alpha[0.9]+)? 
 # in lex syntax; if there is no alpha number, GC_ALPHA_VERSION is empty
 #
-AC_DEFUN(GC_SET_VERSION, [
+AC_DEFUN([GC_SET_VERSION], [
   AC_MSG_CHECKING(GC version numbers)
   GC_VERSION_MAJOR=`echo $PACKAGE_VERSION | sed 's/^\([[0-9]][[0-9]]*\)[[.]].*$/\1/g'`
   GC_VERSION_MINOR=`echo $PACKAGE_VERSION | sed 's/^[[^.]]*[[.]]\([[0-9]][[0-9]]*\).*$/\1/g'`
@@ -46,4 +46,3 @@ AC_DEFUN(GC_SET_VERSION, [
 ${GC_ALPHA_VERSION:+alpha=}$GC_ALPHA_VERSION)
 ])
 
-sinclude(libtool.m4)
diff --git a/doc/Makefile.am b/doc/Makefile.am
index 9446bcb..8f00968 100644
--- a/doc/Makefile.am
+++ b/doc/Makefile.am
@@ -15,7 +15,8 @@
 
 # installed documentation
 #
-dist_pkgdata_DATA = barrett_diagram debugging.html gc.man \
+dist_man3_MANS = gc.man
+dist_pkgdata_DATA = barrett_diagram debugging.html \
     gcdescr.html README README.amiga README.arm.cross \
     README.autoconf README.changes README.contributors \
     README.cords README.DGUX386 README.dj README.environment \
